app/views/admins/orders/_form_search.html.erb

```diff
@@ -1,26 +1,28 @@
-<%= form_tag admins_orders_path, method: :get,class: "form-search-admin" do %>
+<%= search_form_for @q, url: admins_orders_path, html: {method: :get, class: "form-search-admin"} do |f| %>
   <div class="d-flex">
-    <%= search_field_tag :order_id,
-      params[:order_id],
+    <%= f.search_field :id_eq,
       class: "form-control",
       placeholder: t(".order_id") %>
-    <%= search_field_tag :room_id,
-      params[:room_id],
+    <%= f.search_field :room_id_eq,
       class: "form-control",
       placeholder: t(".room_id") %>
-    <%= select_tag :status,
+    <%= f.select :status_eq,
```

Comment:
- `eq` là gì
- `status_eq` hiểu ntn

Reply:
eq là predicate do ransack cung cấp, thể hiện cho toán tử equal
status_eq : Nghĩa là tìm các row có status bằng input ạ